### PR TITLE
HWP-409: Post and post page styling

### PR DESCRIPTION
**Pull request by BradyMitch**
# Description

This PR includes the following proposed change(s):

- Styled posts and post pages to align closely with the new wireframes
- Added published label on posts to display when the post was created
- Published label is converted from utc to the user's local time

## Requires

Add 'Requires Attention' label if appropriate.

- [ ] Requires an update to the .env file
- [ ] Requires adding a file listed in the .gitignore
- [ ] Requires an opinion or answer to a question, see comments.
- [ ] Requires other attention, see below.

If there are any requirements for the developer to make after this PR is merged please list them here.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Tested on homepage and community page.

For pull requests that are not just a simple fix, please check that the branch works on your local machine.

- [ ] Tested by Zach.
- [x] Tested by Brandon.
- [x] Tested by Brady.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes
- [ ] If attention is required by the developer such as updating the .env file, these requirements have been listed.

### CODE REVIEW

File: app/client/src/components/Post.jsx [hunk 1]
```diff
@@ -66,22 +67,52 @@ const Post = (props) => {
...
+
+  if (splitCreatedOn[0] === today) createdOn = `Today${splitCreatedOn[1]}`;
+  if (splitCreatedOn[0] === yesterday)
+    createdOn = `Yesterday${splitCreatedOn[1]}`;
+
   return (
-    <div key={post._id}>
+    <div
+      key={post._id}
+      style={{ "margin-bottom": "15px", "background-color": "transparent" }}
```
Comment (Bottle7) [inline app/client/src/components/Post.jsx:98]: Whenever you want to style components, use the mui sx prop. Using sx is the recommended method to use for styling and it is theme-aware.

**Review by Bottle7 — APPROVED** [review]
Tested. Great job!

File: app/client/src/components/Post.jsx (hunk 1 above)
Comment (BradyMitch) [reply app/client/src/components/Post.jsx:98]: I just thought sx was only for react components. I'll change it to keep consistency.

Comment (BradyMitch) [reply app/client/src/components/Post.jsx:98]: I just tested and sx doesn't work on <div>. We could change the div's to <Box> for now I'll leave it as is.

**Pull request merged**